codespell_lib/data/dictionary.txt

```diff
@@ -12130,6 +12139,7 @@ charictors->characters
 chariman->chairman
 charistics->characteristics
 charizma->charisma
+charnge->change, charge,
```

Comment:
There is the possibility somebody is trying to spell "Charn" (from a fiction book). I assume that's too much of an edge case?